pkg/manifest/manifest_writer.go

```diff
@@ -139,6 +139,7 @@ func getAuth(env EnvironmentDefinition) auth {
 
 	var te *url
 	switch env.Auth.OAuth.TokenEndpoint.Type {
+	case Absent:
```

Comment:
won't that change result in writing out the default as a value after all? 
If the field was absent from, we now treat it like a value and write whatever URL is in it, rather than not writing it

Reply:
Not in go, you'd have to use the `fallthrough`. 
Not sure why we need this change anyway